### ISSUE TITLE
Overlay Opacity settings

### ISSUE THREAD

**Issue by SpikyEchidna** [citra-emu/citra#7185]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### What feature are you suggesting?

Add a setting, preferably a slider, where you can set the overlay opacity

### Why would this feature be useful?

Less distractions during gameplay, better viewing experience

**Comment by Skyrimer3d** [comment]
Really interested in this too pls, it's pretty basic and really helps getting more immersed into games lowering the opacity for a bit.

**Comment by HydeFromT70s** [comment]
Yeah, I vote for this future as well. I was just searching for this option and to my surprise there's no such an option :(
